src/star-rating/index.js

```diff
@@ -0,0 +1,33 @@
+const simpleIcons = require("simple-icons");
+const starSVG = require("./star.svg.js");
+const starRatingComponentStyles = require("./index.style");
+
+const starRating = async ({ skill, text }) => {
+  let content = "";
+  Array.from({ length: 5 }).map((_r, index) => {
+    if (index < parseInt(text)) {
+      content += starSVG("#FFD700", "#FFD700");
```

Comment:
If we were to use the same color for the border as well as fill we don't need two separate params right?

Reply:
If we use the same color for stroke/border instead of two separate params, it will be `none` as well and empty stars  that denote maximum rating allowed is 5. won't appear.  Refer SS below. If this looks fine then we can have only 1 param `color`

Before  : with separate params for stroke and fill
![Screenshot 2022-02-28 at 2 07 21 PM](https://user-images.githubusercontent.com/24493084/156703861-f9e484e5-bfa0-4480-9732-51ad9895e7c9.png)

After: with common param color
![Screenshot 2022-03-04 at 10 24 32 AM](https://user-images.githubusercontent.com/24493084/156703543-3ae024d9-ca1a-4d19-88af-c60bc4a262f2.png)
![Screenshot 2022-03-04 at 10 24 46 AM](https://user-images.githubusercontent.com/24493084/156703553-20a0369f-9097-47d7-adcb-d29b67499c57.png)

Reply:
Should we send only 1 param `color`?

Reply:
Hey sorry ive been caught up with some work will get back soon

Reply:
Can you invite me as collaborator to your repo please? @pranaybinju. I have some code changes I'll commit them

Reply:
sure @harish-sethuraman . I have invited you.

Reply:
Would you like to verify if everything works as expected?

Reply:
Verified. Looks good to me